test/python/api_config.py

```diff
@@ -69,6 +85,7 @@ def setUpClass(cls):
         cls.account = os.environ[CONJUR_ACCOUNT]
 
         cls.client = get_api_client()
+        cls.config = cls.client.configuration
```

Comment:
Not breaking, just seems unnecessary. Just exposing `cls.client.configuration` twice.